app/test_main.py

```diff
@@ -1,3 +1,30 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, result",
+    [
+        (-1, -3, [0, 0]),
```

Comment:
The test case with negative values `(-1, -3, [0, 0])` assumes that the function should return `[0, 0]` for negative inputs. It's important to verify if this behavior is intended. Typically, negative ages might be considered invalid, and the function could raise an exception or handle it differently. Please ensure that this behavior aligns with the intended functionality of `get_human_age`.